### PR TITLE
Feature/pt 3464 cancel looksrare events

### PR DESCRIPTION
**Pull request by dmitry-saraykin**

### CODE REVIEW

File: order/core/src/main/kotlin/com/rarible/protocol/order/core/model/LooksrareV2Cursor.kt [hunk 1]
```diff
@@ -1,13 +1,42 @@
...
+        } else {
+            logger.looksrareInfo("Load all, max createdAfter=$savingMaxSeenCreated")
+            LooksrareV2Cursor(createdAfter = savingMaxSeenCreated)
+        }
+    }
+
     companion object {
+        private val logger = LoggerFactory.getLogger(LooksrareV2Cursor::class.java)
+
         @Deprecated("Remove after migration")
```
Comment (yakischik) [inline order/core/src/main/kotlin/com/rarible/protocol/order/core/model/LooksrareV2Cursor.kt:40]: Is it still needed?

File: order/listener/src/main/kotlin/com/rarible/protocol/order/listener/service/looksrare/LooksrareCancelListEventLoader.kt [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+) {
+    private val logger = LoggerFactory.getLogger(javaClass)
+
+    private val seenEvents = Caffeine.newBuilder()
+        .maximumSize((properties.loadMaxSize * properties.loadMaxDeep).toLong())
+        .build<String, Boolean>()
+
+    suspend fun load(cursor: LooksrareV2Cursor): Result {
+        val result = safeGetNextEvents(cursor)
+        val saved = AtomicInteger(0)
```
Comment (yakischik) [inline order/listener/src/main/kotlin/com/rarible/protocol/order/listener/service/looksrare/LooksrareCancelListEventLoader.kt:36]: Can we do it in async way?

File: order/core/src/main/kotlin/com/rarible/protocol/order/core/model/LooksrareV2Cursor.kt (hunk 1 above)
Comment (dmitry-saraykin) [reply order/core/src/main/kotlin/com/rarible/protocol/order/core/model/LooksrareV2Cursor.kt]: removed

File: order/listener/src/main/kotlin/com/rarible/protocol/order/listener/service/looksrare/LooksrareCancelListEventLoader.kt (hunk 2 above)
Comment (dmitry-saraykin) [reply order/listener/src/main/kotlin/com/rarible/protocol/order/listener/service/looksrare/LooksrareCancelListEventLoader.kt:36]: I don't think it makes much sense. There are not so many offchain events and it's a background job which doesn't affect any pipeline

**Review by yakischik — APPROVED** [review]

**Pull request merged**